package.json

```diff
@@ -57,6 +67,7 @@
     "@vueuse/components": "^10.0.2",
     "@vueuse/core": "^10.1.2",
     "clone": "^2.1.2",
+    "core-js": "^3.31.0",
```

Comment:
Is i really needed?

Reply:
Yes to resolve the polyfills added by the corejs rollup plugin.

(I still think we should not add them in libraries but either globally or the calling app, but that is still to discuss https://github.com/nextcloud/standards/issues/10 )

Reply:
None of the other apps/libs ships core-js
We removed it in the past, why do we want it back? What is different with vite that makes it necessary compared to webpack here?

Reply:
Oh. I really thought our babel config was set to `useBuiltIns: 'usage'`?
Well in this case lets disable polyfilling here :broom: